fabfile/setup.py

```diff
@@ -223,19 +190,17 @@ def existing():
     add_fab_path_to_bashrc()
     git_repo = raw_input("Enter the git repository link\n(i.e. git@github.com:mr_programmer/robot_repository.git):\t")
     project_name = get_project_name_from_repo(git_repo)
+    repo_name = get_project_name_from_repo(git_repo, False)
     env.settings = {
         'git_repo': git_repo,
         'project_name': project_name
     }
     local("git clone {}".format(git_repo))
-
-    execute(create_ansible_env)
+    local("mv ./{} ./{}".format(repo_name, project_name))
     with bash():
         local("cp $FAB_PATH/../orchestration/Vagrantfile ./")
     recursive_file_modify('./Vagrantfile', env.settings, is_dir=False)
-    with ansible():
-        local("vagrant plugin install vagrant-vbguest")
-        local("vagrant up")
+    local("vagrant up")
```

Comment:
Would it be worth while checking somewhere that they have ansible installed and informing them to download it if it isn't? Basically I'm thinking a try/except around importing ansible. Now that we're not attempting to install some things for users, it may be worthwhile to give friendly error messages when they're not installed.